README.md

```diff
@@ -2,6 +2,145 @@
+  and if it did, then whether the PR was labeled with `hra-release`.
+- [git2](https://github.com/rust-lang/git2-rs) to tag the HEAD of the current branch with the new version number, and
+  then to push that tag to the remote repository.
+- The [GitHub CLI](https://cli.github.com/) against to create a GitHub release for the new version.
+
+## Committing to a repository that uses this tool
+
+When committing to a repository that uses this tool, you should follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) 
+format. This will allow the tool to automatically determine the next version number based on the commit history.
+
+To get an idea of the commit messages that `git-cliff` is configured to expect, you can check the `pre-1.0-cliff.toml` 
+file for the `git.commit_parsers` field. These patterns are used to parse commit messages and group them under 
+categorized headings in the generated changelog.
+
+Note that the `cocogitto` bot is enabled on all Holochain repositories, and will automatically reject commits that do
+not follow the Conventional Commits format. However, just because a commit is compliant with Conventional Commits does
+not necessarily mean it will show up how you expect in the changelog. It is preferred that commits use messages which
+fit the patterns mentioned above.
+
+Currently, Holochain repositories permit either "Rebase and merge" or "Squash and merge" as the merge strategy for pull 
+requests. Unless you are working on a pull request that was opened before the introduction of this tool, it is 
+recommended to use "Rebase and merge" as the merge strategy.
+
+## Maintaining repositories that use this tool
+
+There are a few important things to know when maintaining repositories that use this tool:
+- Release tags are filtered by branch so that only tags that are relevant to the current branch are considered. This is 
+  done to permit creating new releases from release branches, after newer versions have been published from the main 
+  branch.
+- Although the tool used to manage versions in `Cargo.toml` files (cargo-workspaces) is capable of understanding various
+  strategies for versioning crates within a workspace, this tool only supports using a single version for all crates and
+  it must be specified in the root `Cargo.toml` file. Use the `[workspace.package]` section to specify the version and 
+  then reference that version in crates as `version.workspace = true`.
+
+## Integrating the tool into a repository
+
+This repository only provides a binary CLI tool, which is published to a release on this repository when the repository
+is tagged. To integrate the tool, you need to workflows which are provided in the `holochain/actions` repository.
+
+The release [preparation workflow](https://github.com/holochain/actions/blob/main/.github/workflows/prepare-release.yml)
+can be added to your repository with a workflow that looks like this:
```

Comment:
Do we intend to keep the readme here in lockstep with the real file in the `holochain/actions` repository? Otherwise we might want to add an explicit hint that the `holochain/actions` repo is the source of truth.

Reply:
Yep, that would be a useful note about versioning in general, added